sources/us/il/woodford.json

```diff
@@ -0,0 +1,26 @@
+{
+    "coverage": {
+        "US Census": {
+            "geoid": "17203",
+            "name": "Woodford County",
+            "state": "Illinois"
+        },
+        "country": "us",
+        "state": "il",
+        "county": "Woodford"
+    },
+    "data": "http://www.centralilmaps.com/arcgis/rest/services/WoodfordGIS_Flex/MapServer/11",
+    "type": "ESRI",
+    "conform": {
+        "type": "geojson",
+        "number": "ADDR_HN",
+        "street": [
+          "ADDR_PD",
+          "ADDR_PT",
+          "ADDR_ST_NA",
+          "ADDR_ST",
```

Comment:
I'm afraid the extra comma here makes this invalid JSON which is causing the tests to fail.